gallery/src/PageHeader.tsx

```diff
@@ -16,12 +16,10 @@ function PageHeader() {
       <div className="nx-page-header__inner">
         <a href="#" className="nx-product">
           <img src={sonatypeLogo} className="nx-product__logo"/>
-          <div className="nx-product__name">
-            Sonatype Component Library
-          </div>
-          <div className="nx-product__version">
-            Version: {packageJson.version}
-          </div>
+          <div className="nx-product__sonatype-wordmark">sonatype</div>
+          <div className="nx-product__vertical-divider">|</div>
+          <div className="nx-product__name">react shared component library</div>
+          <div className="nx-product__version">Version: {packageJson.version}</div>
```

Comment:
Should these be spans at this point?

Reply:
> Should these be spans at this point?

They could be I guess. Since we're using flex box I don't see much reason to switch them